bob/src/api/http.rs

```diff
@@ -568,13 +584,23 @@ async fn alien(_bob: &State<BobServer>) -> &'static str {
     "alien"
 }
 
+#[post("/alien/sync")]
+async fn sync_alien_data(bob: &State<BobServer>) -> Result<StatusExt, StatusExt> {
```

Comment:
Ambiguous route an fn names. It detaches all alien holders, why don you named it like this?

Reply:
From the user point of view it just syncs alien data with disk. I can change it to `alien/detach`